### PR TITLE
issue: external storage naming #11755

### LINKED ISSUE #11755: external storage naming
Please add validation for adding external storage:
name should not have '/' symbol.

I've added, but I can't use it. Well, webplayer somehow get known about music, but other parts (wev, android app) can't see it.

### PR DESCRIPTION
**Pull request by MasterOfDeath**
Validation for existing mountpoint path
fixes #11755

### CODE REVIEW

File: apps/files_external/l10n/en_GB.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "Storage with ID \"%d\" not found" : "Storage with ID \"%d\" not found",
     "Invalid backend or authentication mechanism class" : "Invalid backend or authentication mechanism class",
     "Invalid mount point" : "Invalid mount point",
+    "Invalid path of mountpoint" : "Invalid path of mountpoint",
```
Comment (rullzer) [inline apps/files_external/l10n/en_GB.json]: This is not required. Our translation scrips sync this auytmaitcally to transifex for translations

Comment (MasterOfDeath) [reply apps/files_external/l10n/en_GB.json]: Should I remove it from PR?

Comment (MorrisJobke) [reply apps/files_external/l10n/en_GB.json]: Yes - the automatic script will update it and the translations are stored on transifex anyways.

File: apps/files_external/l10n/en_GB.js [hunk 2]
```diff
@@ -45,6 +45,7 @@ OC.L10N.register(
     "Storage with ID \"%d\" not found" : "Storage with ID \"%d\" not found",
     "Invalid backend or authentication mechanism class" : "Invalid backend or authentication mechanism class",
     "Invalid mount point" : "Invalid mount point",
+    "Invalid path of mountpoint" : "Invalid path of mountpoint",
```
Comment (MorrisJobke) [inline apps/files_external/l10n/en_GB.js]: Same for this.

**Comment by MorrisJobke** [comment]
@icewind1991 I would like to have your feedback on this one.

**Comment by rullzer** [comment]
@icewind1991 ping

**Comment by rullzer** [comment]
I'm going to clos this here.
@icewind1991 if you still want this please reopen.

**Pull request closed without merging**